.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ workflows:
           name: "package and push linkerd2 chart"
           app_catalog: "giantswarm-playground-catalog"
           app_catalog_test: "giantswarm-playground-test-catalog"
-          chart: "linkerd2-app"
+          chart: "linkerd2"
```

Comment:
CI fails because the chart must start with the repo name.

> chart parameter value must start with linkerd2-app

I'd like us to make an exception for apps but still at the idea stage.